scipy/signal/wavelets.py

```diff
@@ -467,7 +467,8 @@ def cwt(data, wavelet, widths, dtype=None, **kwargs):
 
     output = np.empty((len(widths), len(data)), dtype=dtype)
     for ind, width in enumerate(widths):
-        N = np.min([10 * width, len(data)])
+        # Force N to be odd to avoid shifting the peak
+        N = int(np.min([10 * width, len(data)])) // 2 * 2 + 1
```

Comment:
I would not touch this and use a modulo operation instead.
```suggestion
        N = np.min([10 * width, len(data)])
        # Force N to be odd to avoid shifting the peak
        N += 1 - N % 2
```

Reply:
Or `N += (N+1) % 2`